electricitylci/modelconfig/ELCI_1_config.yml

```diff
@@ -85,7 +85,7 @@ NETL_IO_trading_year: 2016
 
 # OTHER PARAMETERS
 
-fedelemflowlist_version: 1.0.1
+fedelemflowlist_version: 1.0.3
```

Comment:
@bl-young  v1.0.3 has not ever been released...this needs to be v1.0.2 in order to match the requirements

Reply:
Ok this gets to your email regarding the release of v1.0.3, but I recognize currently the setup requirements only list v1.0.2
So I will change this to v1.0.2.

Reply:
@bl-young OK v1.0.2 is good